controllers/loadtest_controller.go

```diff
@@ -139,12 +155,39 @@ func (r *LoadTestReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		}
 	}
 
+	reQueueTime := getReQueueTime(test, previousStatus, log)
```

Comment:
I hope this doesn't sound difficult, but I prefer the `requeueTime` and `getRequeueTime` casing, since it matches `ctrl.Result{Requeue: true}` and `ctrl.Result{RequeueAfter: ...}`.

Reply:
No, you are helping find things I missed, I appreciate that! I fix it after the meeting.:) I also like to keep it tidy.